swagger.json

```diff
@@ -88,95 +97,274 @@
         }, {
           "in" : "body",
           "name" : "body",
-          "description" : "JSON-LD Representation of the resource metadata you want to change for that specified resouce (identified via its identifier or DRUID). Needs to fit the SDR 3.0 MAP requirements.",
+          "description" : "JSON-LD Representation of the resource metadata required fields and only the fields you wish to update (identified via its TACO identifier). Needs to fit the SDR 3.0 MAP requirements.",
           "required" : true,
           "schema" : {
             "$ref" : "#/definitions/Resource"
           }
         } ],
         "responses" : {
           "200" : {
-            "description" : "OK"
+            "description" : "TACO resource metadata updated & processing started.",
+            "schema" : {
+              "$ref": "#/definitions/ResourceResponse"
+            }
           },
           "400" : {
             "description" : "Invalid ID supplied"
           },
-          "404" : {
-            "description" : "Resource not found"
+          "401" : {
+            "description" : "You are not authorized to update a resource in TACO."
+          },
+          "415" : {
+            "description" : "Unsupported resource type provided. TACO resources should be handed over as JSON or JSON-LD."
+          },
+          "422" : {
+            "description" : "The resource JSON provided had an unspecified or unsupported field, or is otherwise unprocessable by TACO."
           },
-          "405" : {
-            "description" : "Validation exception"
+          "500" : {
+            "description" : "This resource could be updated at this time by TACO."
           }
         }
       }
     },
     "/file" : {
       "post" : {
-        "summary" : "Deposit a new File (binary) into SDR.",
+        "summary" : "Deposit New File (binary).",
         "description" : "Deposits a new File (binary) into SDR. Will return the SDR identifier for the File resource (aka the metadata object generated and persisted for File management).",
         "operationId" : "depositNewFile",
         "produces" : [ "application/json" ],
         "parameters" : [ {
           "in" : "body",
           "name" : "body",
-          "description" : "File / Binary.",
+          "description" : "Binary to be added to an Object in TACO.",
           "required" : true,
           "schema" : {
             "$ref" : "#/definitions/File"
```

Comment:
Was this what we talked about today?  Shouldn't this be a binary object?

Reply:
it is.

Reply:
Doesn’t this point to 293 which indicates it’s a JSON object?